torchtune/modules/transformer.py

```diff
@@ -767,20 +774,20 @@ def setup_caches(
             )
 
     @property
-    def encoder_caches_are_enabled(self) -> bool:
+    def encoder_caches_are_setup(self) -> bool:
```

Comment:
cc @pbontrager, but I don't know if this would ever be useful.

Reply:
It doesn't make sense to have an encoder cache. If we ever need caches in the encoder (which we never need at the moment), then the fusion model or fusion layers would be responsible for calling the encoder in their caches_are_setup property.

Reply:
This function doesn't strictly check if the encoder has a cache setup, but rather any crossattention layers exist with caches setup, which is useful for input validation during inference with fusion models.